### ISSUE TITLE
VSCODE

### ISSUE THREAD

**Issue by daplcor** [kadena-community/pact-vscode#5]
Installing from scratch gives various errors that need to be worked through when installing/compiling with vsce. It may also be helpful to introduce better instructions on how to initialize the lsp-server and connect to it with Vscode.  Are there any plans to publish this to the extension marketplace for average users who don't want to compile custom extensions?

**Comment by jessevanmuijden** [comment]
Yes, @daplcor, we are working on publishing the latest update of the extension.

**Comment by ra-sun-god** [comment]
I have created a pull request to add a command which will build the source into .vsix, once merged, you can run:

`npm run build`

pull request  [here](https://github.com/kadena-community/pact-vscode/pull/9)
